### PR TITLE
fix: rename project folder in Bento

### PR DESCRIPTION
**Pull request by parano**
## Checklist:

- [ ] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

**Review by sauyon — APPROVED** [review]
LGTM, should I just merge stuff if it looks good?

**Comment by parano** [comment]
@sauyon I was gonna wait for the CI to be fixed but yes let's merge the pending PRs if they look good, I will go back and fix tests if the PR actually breaks anything

**Comment by sauyon** [comment]
Ah, right, that seems sensible to me.

**Pull request merged**
